app/views/missing/report.php

```diff
@@ -195,7 +195,7 @@
                                     </div>
                                     <div class="panel-body" id="panelBody4">
                                         <label for="w_phone_no">Phone number </label>
-                                        <input id="w_phone_no" class="form-control" placeholder="Enter Phone Number" name="contact_whom[phone_no]" required /><br>
+                                        <input id="w_phone_no" class="form-control" placeholder="Enter Phone Number" name="contact_whom[phone_no]" pattern="\d{10}" required /><br>
```

Comment:
Shouldn't we allow an optional `+` at beginning and length from 10 to 15 digits? (Not everywhere in world we have number of exactly 10 digits)

Also possibly allowing white spaces and brackets and dashes?